larreco/HitFinder/GausHitFinder_module.cc

```diff
@@ -102,30 +109,32 @@ class GausHitFinder : public art::EDProducer {
 
 //-------------------------------------------------
 //-------------------------------------------------
-GausHitFinder::GausHitFinder(fhicl::ParameterSet const& pset)
-  : EDProducer{pset},
-    fEventCount(0)
-{
-    fCalDataModuleLabel  = pset.get< std::string >("CalDataModuleLabel");
-    fAllHitsInstanceName = pset.get< std::string >("AllHitsInstanceName","");
-    fFilterHits          = pset.get< bool        >("FilterHits",false);
-    fFillHists           = pset.get< bool        >("FillHists",false);
+GausHitFinder::GausHitFinder(fhicl::ParameterSet const& pset, art::ProcessingFrame const&)
+  : SharedProducer{pset},
+    fFilterHits(pset.get< bool>("FilterHits",false)),
+    fFillHists(pset.get<bool>("FillHists",false)),
+    fCalDataModuleLabel(pset.get< std::string >("CalDataModuleLabel")),
+    fAllHitsInstanceName(pset.get< std::string >("AllHitsInstanceName","")),
+    fLongMaxHitsVec(pset.get< std::vector<int>>("LongMaxHits",    std::vector<int>() = {25,25,25})),
+    fLongPulseWidthVec(pset.get< std::vector<int>>("LongPulseWidth", std::vector<int>() = {16,16,16})),
+    fMaxMultiHit(pset.get< int             >("MaxMultiHit")),
```

Comment:
Are the default values needed?
I'd imagine they can only be relevant for one experiment. 
If an experiment misconfigures their fcl file then that experiment would be exposed to these default values.